CI/appveyor.install.sh

```diff
@@ -22,7 +22,7 @@ if [ ${BUILD_BITNESS} != "32" ] && [ ${BUILD_BITNESS} != "64" ] ; then
     exit 1
 fi
 
-appveyor AddMessage "This is a \"${BUILD_TYPE}\" ${BUILD_BITNESS} bit Mudlet ${VERSION}${MUDLET_VERSION_BUILD} build ..."" -Catagory Information
+appveyor AddMessage "This is a \"${BUILD_TYPE}\" ${BUILD_BITNESS} bit Mudlet ${VERSION}${MUDLET_VERSION_BUILD} build ..." -Catagory Information
```

Comment:
Category?

Reply:
It is an argument to the "AddMessage" (displays a text on the AppVeyor build "Messages" tab) - with one of three values:
* Information
* Warning
* Error
see https://www.appveyor.com/docs/build-worker-api/#add-compilation-message ...

Reply:
Yeah but you spelled it as `Catagory`.

Reply:
🤦‍♂️